packages/lit-html/src/test/polyfills.ts

```diff
@@ -0,0 +1,41 @@
+
+(function (arr) {
+  arr.forEach(function (item) {
+    if (item.hasOwnProperty('remove')) {
+      return;
+    }
+    Object.defineProperty(item, 'remove', {
+      configurable: true,
+      enumerable: true,
+      writable: true,
+      value: function remove() {
+        this.parentNode.removeChild(this);
+      }
+    });
+  });
+})([Element.prototype, CharacterData.prototype, DocumentType.prototype]);
+
+// Source: https://github.com/jserz/js_piece/blob/master/DOM/ParentNode/append()/append().md
```

Comment:
Link 404'ed; just wanted to double-check license.

Reply:
That link was copied with the code from MDN. I'll link to MDN instead.